doc/configuration.md

```diff
@@ -760,9 +760,9 @@ This setting will be ignored if no [key][key] has been provided.
 ## Metadata (`metadata`)
 
 The metadata (`any` default none) setting can be any value. This value will be stored as metadata that can be retrieved
-from the built PHAR ([`Phar::getMetadata()][phar.getmetadata]).
+from the built PHAR ([Phar::getMetadata()][phar.getmetadata]).
```

Comment:
Since this is referencing code, how about wrapping this in backticks?

```suggestion
from the built PHAR ([`Phar::getMetadata()`][phar.getmetadata]).
```

Reply:
No, the backticks was introduced by error in my previous contribution. I've fixed it but didn't realized until now, that there is again a new error in anchor link at bottom of this document (protocol error)
https://github.com/llaville/box/blame/build-metadata/doc/configuration.md#L1007

Reply:
fixed with commit 38b5c3ef